### PR TITLE
 [networkx] Support label on DynamicFragment

### LINKED ISSUE #804: [NetworkX] support label on DynamicFragment
- support vertex label on DynamicFragment
- revise the arrow_to_dynamic to preserve label

### PR DESCRIPTION
**Pull request by acezen**
## What do these changes do?
- support label on DynamicFragment
- revise arrow_to_dynamic_fragment to preserve label of property graph
- support `(label, id)` node layout in nx.Graph and nx.DiGraph

## Related issue number
Fixes #804 
related #806 

## ToDo
- [x] update vineyard image
- [x] add test case

### CODE REVIEW

File: python/graphscope/nx/classes/digraph.py [hunk 1]
```diff
@@ -219,6 +220,10 @@ def __init__(self, incoming_graph_data=None, **attr):
             "create_empty_in_engine", True
         )  # a hidden parameter
         self._distributed = attr.pop("dist", False)
+        if incoming_graph_data is not None and self._is_gs_graph(incoming_graph_data):
+            # convert from gs graph always use distributed mode
+            self._distributed = True
+        self._default_label = attr.pop("default_label", "_")
```
Comment (sighingnow) [inline python/graphscope/nx/classes/digraph.py:226]: Add the `default_label` explicitly to the parameters.

File: python/graphscope/nx/tests/test_nx.py [hunk 2]
```diff
@@ -356,11 +369,6 @@ def test_error_on_wrong_nx_type(self):
         with pytest.raises(TypeError):
             nx_g = nx.DiGraph(g)
 
-    def test_error_on_duplicate_oid(self):
-        g = self.duplicated_oid_g
-        with pytest.raises(AnalyticalEngineInternalError):
-            nx_g = self.NXGraph(g)
```
Comment (sighingnow) [inline python/graphscope/nx/tests/test_nx.py:362]: Why we drop this test case?

File: python/graphscope/nx/tests/test_nx.py [hunk 3]
```diff
@@ -323,28 +312,52 @@ def test_error_on_mixing_node_nx_to_gs(self):
...
+        assert ("post", 618475290624) in nx_g
+        nx_g2 = self.NXGraph(g, default_label="comment")
+        assert nx_g2.number_of_nodes() == (76830 + 903 + 78976)
+        assert nx_g2.number_of_edges() == (38786 + 6626 + 38044)
+        assert 618475290625 in nx_g2  # comment node is default label node
+        assert ("comment", 618475290625) not in nx_g2
+        assert 933 not in nx_g2  # person node is (label, id) format
+        assert ("person", 933) in nx_g2
+        assert 618475290624 not in nx_g2  # post node is (label, id) format
+        assert ("post", 618475290624) in nx_g2
```
Comment (sighingnow) [inline python/graphscope/nx/tests/test_nx.py:352]: Could you please add a test case where `default_label` is involved as well?

**Review by sighingnow — APPROVED** [review]
Overall LGTM.

File: python/graphscope/nx/tests/test_nx.py (hunk 2 above)
Comment (acezen) [reply python/graphscope/nx/tests/test_nx.py:362]: since we add label support, there would be no duplicated oid.

**Pull request merged**